Makefile.defs

```diff
@@ -836,16 +836,21 @@ TURBO_BUILD_DIR=$(THIRDPARTY_DIR)/turbo/build/$(MACHINE)
 TURBO_DIR=$(CURDIR)/$(TURBO_BUILD_DIR)/turbo-prefix/src/turbo
 TURBO_FFI_WRAP_LIB=$(OUTPUT_DIR)/common/libtffi_wrap$(LIB_EXT)
 
-LUA_SPORE_BUILD_DIR=$(THIRDPARTY_DIR)/lua-Spore/build/$(MACHINE)
-LUA_SPORE_DIR=$(CURDIR)/$(LUA_SPORE_BUILD_DIR)/lua-Spore-prefix/src/lua-Spore
-SPORE_VER=0.3.1-1
-LUA_SPORE_VER=lua-spore-$(SPORE_VER)
 # Unversioned LR 2 path
 LUAROCKS_DIR=$(OUTPUT_DIR)/rocks/lib/luarocks/rocks
 # Versioned LR 3 path, see koreader-base/pull/1003
 ifeq ($(LR_GT_3_0),1)
 LUAROCKS_DIR=$(OUTPUT_DIR)/rocks/lib/luarocks/rocks-5.1
 endif
+
+LUA_HTMLPARSER_VER=scm-0
```

Comment:
Or fix it here instead, but I kinda prefer the other approach myself ;).

(i.e., VER being a version only, and prepending the package name in the CMake file).

In which case, please tweak lua-Spore, too, for consistency's sake ;).